include/config.h

```diff
@@ -0,0 +1,11 @@
+#ifndef CONFIG_H
+#define CONFIG_H
+
+/**
+ * @brief Use a custom course engine
+ * Use a custom course engine instead of the default one who use switch. see course.h and everywhere
+ * USE_CUSTOM_COURSE_ENGINE are use to define your
+ *
+ */
+#define USE_CUSTOM_COURSE_ENGINE 0
```

Comment:
`CUSTOM_COURSE_ENGINE`

Reply:
I prefer add USE to say it will use CUSTOM COURSE ENGINE and he didn't define the CUSTOM COURSE ENGINE

Reply:
None of the other defines have 'use' or anything of the sort.

Reply:
Hmm I can change to ENABLE_CUSTOM_COURSE_ENGINE